gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/service/TaskDtoBuilderService.java

```diff
@@ -104,46 +108,47 @@ public List<TaskDto> getListRunningTasksDto() {
                 .toList();
     }
 
-    public TaskDto getEmptyTask(OffsetDateTime timestamp) {
+    public TaskDto getEmptyTask(final OffsetDateTime timestamp) {
         return TaskDto.emptyTask(
                 timestamp,
                 properties.getProcess().getInputs(),
                 properties.getProcess().getOutputs());
     }
 
-    public TaskDto createDtoFromEntity(Task task) {
-        return createDtoFromEntityWithOrWithoutEvents(task, true);
+    public TaskDto createDtoFromEntity(final Task task) {
+        return createDtoFromEntityGivenEvents(task, true);
     }
 
-    public TaskDto createDtoFromEntityWithoutProcessEvents(Task task) {
-        return createDtoFromEntityWithOrWithoutEvents(task, false);
+    public TaskDto createDtoFromEntityWithoutProcessEvents(final Task task) {
+        return createDtoFromEntityGivenEvents(task, false);
     }
 
-    private TaskDto createDtoFromEntityWithOrWithoutEvents(Task task, boolean withEvents) {
+    private TaskDto createDtoFromEntityGivenEvents(final Task task, final boolean withEvents) {
+
+        final Function<String, Stream<ProcessFileDto>>mapInputFile = str ->  task.getInput(str)
+                                                                            .stream()
+                                                                            .map(this::createDtoFromEntity);
+
+        final Function<String, ProcessFileDto> mapEmptiableInputFile = str ->  task.getInput(str)
```

Comment:
Not sure I understood well the objective of this function given its name. First I thought that the input file returned by the function could be emptied later, but that's not the supposed meaning I guess.